plugins/workspace-minimap/src/minimap.ts

```diff
@@ -45,7 +53,11 @@ export class Minimap {
      * Initialize.
      */
     init(): void {
+      this.minimapWorkspace.scrollbar.setContainerVisible(false);
       this.primaryWorkspace.addChangeListener((e) => void this.mirror(e));
+      window.addEventListener('resize', (e) => {
```

Comment:
It might be good to debounce this, e.g. only zoom to fit 250ms or so after the last resize event; given that a minimap is most useful when the workspace is large, there might be a noticeable performance penalty from doing this live if the workspace has hundreds of blocks since this callback gets invoked more or less continuously during a window resize. Could you see if there is a noticeable performance impact either way if the workspace has > 100 blocks?

Reply:
Sounds good, I implemented debounce. As for checking if there is a performance impact, I was planning on leaving that for a _next steps_. I tried to quickly create a script to auto-populate the workspace but could not replicate the 'airstrike' function from core.

Reply:
Sorry, I should have been clearer - I think we need to verify the performance impact first, because if the non-debounced implementation is fine, that's preferable, since it updates the state of the minimap faster and requires less/simpler code. Feel free to ping me if you need help setting up something akin to the airstrike functionality!

Reply:
Ok, I was able to generate a script to auto populate the workspace. I tested with anywhere between 100 and 500 blocks at once, and the difference between live and debouncing was marginal. Not only was the performance basically the same, but with the live rendering it looked a lot smoother. I'm going to go ahead and revert to live rendering. Let me know if I should change something else!

Reply:
Sounds good, thanks for checking and updating accordingly!